components/automate-ui/src/app/pages/+compliance/+node-credentials/create-node-credential-modal/create-node-credential-modal.component.spec.ts

```diff
@@ -39,18 +40,18 @@ describe('CreateNodeCredentialModalComponent', () => {
     }).compileComponents();
     store = TestBed.inject(Store);
     spyOn(store, 'dispatch').and.callThrough();
-  });
+  }));
 
-  beforeEach(() => {
+  beforeEach( () => {
     fixture = TestBed.createComponent(CreateNodeCredentialModalComponent);
     component = fixture.componentInstance;
     component.createNodeCredForm = new FormBuilder().group({
       id: [''],
       name: [''],
       type: [''],
-      ssh: this.sshForms,
-      winrm: this.winrmForms,
-      sudo: this.sudoForms
+      ssh: component.sshForms,
+      winrm: component.winrmForms,
+      sudo: component.sudoForms
```

Comment:
This was a setup that caused the following three tests to be broken.  The upgrade helped pick this up and allowed the following three tests to pass.